src/UserNSRunner.jl

```diff
@@ -40,6 +40,18 @@ function platform_def_mapping(platform)
     return mapping
 end
 
+"""
+    ldso_cmd(cmd::Cmd)
+
+We ship on a variety of platforms.  Some of them may not natively share the
+binary loader our sandbox is linked against.  But that's okay, because we ship
+our own.  So we'll just manually invoke it.
+"""
+function ldso_cmd(cmd::Cmd)
```

Comment:
Ideally the sandbox would be statically linked.

Reply:
Great idea.  Didn't realize that static linking would get rid of even the interpreter.